cql2pgjson/src/test/resources/compoundIndex.sql

```diff
@@ -18,7 +18,20 @@ $func$  LANGUAGE sql IMMUTABLE;
 create or replace function concat_space_sql(VARIADIC text[])
 RETURNS text AS $$ select concat_ws(' ', VARIADIC $1); $$ LANGUAGE SQL IMMUTABLE;
 
+create or replace function concat_array_object_values(jsonb_data text, field text) RETURNS text AS $$
```

Comment:
Can you change `jsonb_data text` to `jsonb_data jsonb` to avoid converting the jsonb object array to a string?